DEVELOPMENT.md

```diff
@@ -202,3 +202,48 @@ Additional information which is really important to how the application runs can
 be found in the manifest:
 
 > `$ bundle exec 3scale_backend manifest`
+
+## Running with RubyMine
+
+Using RubyMine scripts can help you run both Worker and Listener.
+
+For that, you'll need to create the following configurations: 
+
+### Worker
```

Comment:
For the worker we can also mention that the sync/async mode is configured by the `CONFIG_REDIS_ASYNC` in `.env`.